CraftTweaker2-MC1120-Main/src/main/java/crafttweaker/mc1120/brewing/MultiBrewingRecipe.java

```diff
@@ -0,0 +1,80 @@
+package crafttweaker.mc1120.brewing;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import crafttweaker.api.item.IIngredient;
+import crafttweaker.api.item.IItemStack;
+import crafttweaker.api.minecraft.CraftTweakerMC;
+import net.minecraft.item.ItemStack;
+import net.minecraftforge.common.brewing.IBrewingRecipe;
+
+public class MultiBrewingRecipe implements IBrewingRecipe {

```

Comment:
Custom Brewing Recipe Class
Can accept an IIngredient as input (as long as the input item matches with any of the ingredient's items, it's fine) and an IIngredient[] as ingredients, because why not